db/index.js

```diff
@@ -7,7 +7,6 @@ function databaseSettings() {
   let logging;
   if(process.env.NODE_ENV === "test") {
     dialect = "sqlite";
-    dbPath = path.resolve("db/database_test.sqlite3");
```

Comment:
Change the testing database from a file to memory. That way, no need to manually remove the testing db file in case something goes wrong.